### ISSUE TITLE
JSON parsing issue in v5.3.0

### ISSUE THREAD

**Issue by aschlumpf** [avens19/winston-sumologic-transport#27]
When upgrading to v5.3.0, I noticed that whenever logs are sent in bulk to Sumo, they are not being parsed into individual JSON logs as they are in v5.2.1. 

I discovered a different issue which is related to the upgrade of the `axios` package, will open a separate issue for that, but not clear what change causes this issue.

To reproduce the issue, I logged a sample JSON using this package once, and in a loop 5 times. 

```TypeScript
  const logData = {
    id: 100,
    name: 'John',
    jobs: [
      {
        id: 1,
        name: 'Engineer',
      },
      {
        id: 2,
        name: 'Architect',
      },
    ],
    customFields: {
      pet: {
        name: 'Skip',
      },
    },
  };
  // Log a single payload
  sumoLogger.info({
      project: 'My Project',
      type: 'Request',
      direction: 'incoming',
      log: logData,
  });
  // Wait for the log to processed in a single request
  sleep(5000);
 // Send the same payload in a loop so that it is batched
  for (let i = 0; i < 5; i++) {
   // The winston `Logger` using this package as a transport
    sumoLogger.info({
      project: 'My Project',
      type: 'Request',
      direction: 'incoming',
      log: logData,
    });
  }
```

The query I am using is simply...
```
_sourceCategory="my/source/category" "My Project"
```

When I view these logs in my Sumo Logic UI, only the single log is parsed as a JSON. The batched request is shown as raw text. 
![image](https://github.com/avens19/winston-sumologic-transport/assets/23568367/0153784e-84d9-4d35-a923-030c4bf8739b)

If I downgrade to v5.2.1, then both of these logs are properly parsed. I can manipulate the query to parse the raw text as json, but that would change how my logs are queried for before the change.

**Comment by aovens-quantifi** [comment]
Thanks for filing! I'll take a look today

**Comment by avens19** [comment]
I was actually unable to repro with your example code but I managed to repro if I explicitly sent content type 'application/json' so I'm now explicitly sending 'text/plain'. If you don't mind trying again with `5.3.1` that would be awesome

**Comment by aschlumpf** [comment]
Thanks for the quick support. 

It looks like the change isn't coming through on 5.3.1. I installed it into my existing module, and a new one which is empty, and the source code does not include the new header. Not sure if you are seeing the same...

That being said, it looks like sending that header fixes it. I implemented it the change inside the dependency source code, and both the single and batched logs parsed as expected!
